content/manager_webui/maintenance.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: bt_wiki
+title: Maintenance Mode
+category: Manager
+draft: false
+abstract: Cloudify's Web Interface
+weight: 200
+---
+
+Managing maintenance mode page will be found at the settings section in the Web UI. Upon routing you will be presented with the manager maintenance mode status and a button used to change it.<br/>
+![maintenance mode page]({{< img "ui/maintenance/ui-maintenance-mode-page.png" >}})
+
+
+Pressing the button to change the maintenance mode status opens a confirmation dialog.<br/>
+![confirmation dialog]({{< img "ui/maintenance/ui-maintenance-confirmation-dialog.png" >}})
+
+
+Confirming makes the manager start entering requested mode, a message will be shown notifying all users.<br/>
```

Comment:
On confirmation, the manager enters the requested mode and a message is shown to any user currently using the UI.